k8s-artifacts/mg-adapter/adapter-key-configmap.yaml

```diff
@@ -0,0 +1,52 @@
+# --------------------------------------------------------------------
+# Copyright (c) 2020, WSO2 Inc. (http://wso2.com) All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# -----------------------------------------------------------------------
+
+# Config map containing the private key for adapter
+
+apiVersion: v1
+kind: ConfigMap
```

Comment:
Lets make this a k8s secret

Reply:
Fixed in 6b6708a8472501213c00230342810f801d5df4cf